ICAFIX/scripts/computeRecleanFeatures.m

```diff
@@ -0,0 +1,285 @@
+function computeRecleanFeatures(StudyFolder, ...
+                                Subject, ...
+                                fMRIListName, ...
+                                RunsXNumTimePoints, ...
+                                hp, ...
+                                Resolution, ...
+                                CorticalParcellationFile, ...
+                                SubRegionParcellationFile, ...
+                                WMLabelFile, ...
+                                CSFLabelFile)
+
+HCPPIPEDIR = getenv('HCPPIPEDIR')
+
+fMRINames = myreadtext(fMRIListName);
+RunsXNumTimePoints = str2double(RunsXNumTimePoints);
+ResolutionNum = str2double(Resolution);
+
+CorticalParcellation=ciftiopen(CorticalParcellationFile,'wb_command');
+
+VisualAreas=[1 2 3 4 5 6 7 13 16 17 18 19 20 21 22 23 48 49 142 152 153 154 156 158 159 160 163 181 182 183 184 185 186 187 193 196 197 198 199 200 201 202 203 228 229 322 332 333 334 336 338 339 340 343];
```

Comment:
Consider using a text file and `-cifti-label-import` with `-discard-others` to do this using files, rather than hardcoding it in matlab (which needs compiling for use without matlab licensing issues).

Reply:
I tried to create a dlabel file using only these visual regions, but when I combine the two hemispheres together into a single dlabel, since the ROIs don't contain the rest of regions, the numbers are not matching with the 360 ROIs... 

How about I made a txt file to save the visual ROI names or just save the above parcel indices?

Reply:
I see your comment below. I will make a HCP-MMP1.0 based visual ROI txt for the script to load.